lib/trino-parquet/src/main/java/io/trino/parquet/ParquetMetadataConverter.java

```diff
@@ -155,6 +156,7 @@ public static LogicalTypeAnnotation getLogicalTypeAnnotation(LogicalType type)
                 yield timestampType(timestamp.isAdjustedToUTC, convertTimeUnit(timestamp.unit));
             }
             case UUID -> uuidType();
+            case FLOAT16 -> float16Type();
```

Comment:
Should this be test covered?

cc @Fokko

Reply:
Extracted to https://github.com/trinodb/trino/pull/21861